benefits/enrollment_switchio/views.py

```diff
@@ -33,10 +34,17 @@ def get_context_data(self, **kwargs):
                 "cta_button": "tokenize_card",
                 "enrollment_method": models.EnrollmentMethods.DIGITAL,
                 "transit_processor": {"name": "Switchio", "website": "https://switchio.com/transport/"},
+                "locale": self._get_locale(request.LANGUAGE_CODE),
             }
         )
         return context
 
+    def _get_locale(self, django_language_code):
+        """Given a Django language code, return the corresponding locale to use with Switchio's tokenization gateway."""
+        # mapping from Django's I18N LANGUAGE_CODE to Switchio's locales
+        locale = {"en": "en", "es": "es"}.get(django_language_code, "en")
```

Comment:
Shoot, their Spanish is code is slightly different from Littlepay's? I think it's fine to dupe this code for each, for now.